elivepatch_server/resources/dispatcher.py

```diff
@@ -65,11 +66,10 @@ def get(self):
         print("get livepatch: " + str(args))
         # check if is a valid UUID request
         args['UUID'] = check_uuid(args['UUID'])
-        uuid_dir = get_uuid_dir(args['UUID'])
-
-        livepatch_full_path = os.path.join(uuid_dir, 'kpatch-main.ko')
+        livepatch_saving_folder = os.path.join('/tmp/','livepatch',args['UUID'])
```

Comment:
Please avoid repeatedly constructing paths by hand in separate methods. Instead of doing this, use `get_uuid_dir`, or assign it to a class member in the constructor.